README.md

```diff
@@ -1,2 +1,265 @@
+
+```sh
+  ruby file-name.rb
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+👤 **Jules Edozie**
+
+- GitHub: [@githubhandle](https://github.com/julzedz)
+- Twitter: [@twitterhandle](https://twitter.com/julzedz)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/julesedozie/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Decorators**
+- [ ] **Modules**
+- [ ] **Test Cases**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/julzedz/oop-school-library/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project please give it a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thank you Microverse!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **Explain About Class Variable And Global Variable?**
+
+  - A class variable starts with an @@ sign which is immediately followed by upper or lower case letter. You can also put some name characters after the letters which stand to be a pure optional. A class variable can be shared among all the objects of a class. A single copy of a class variable exists for each and every given class.
+
+To write a global variable you start the variable with a ``$``sign which should be followed by a name character. Ruby defines a number of global variables which also include other punctuation characters such as ``$_`` and ``$-k``.
+
+
+- **Mention What Is The Difference Between A Gem And A Plugin In Ruby?**
+
+  - **_Gem_**: A gem is a just ruby code. It is installed on a machine, and it’s available for all ruby applications running on that machine.
+  - **_Plugin_**: Plugin is also ruby code, but it is installed in the application folder and only available for that specific application.
+
+- **Explain the role of modules and mixins in Ruby**
+
+  - Modules are Ruby’s way of ``grouping methods``, ``classes``, and constants together to provide a namespace for preventing name clashes. The second purpose of modules is to use them as ``mixins``. Technically, Ruby only supports single inheritance, but by using modules as mixins, it is possible to share code among different classes—a key advantage of multiple inheritance—without having to give up the simplicity of the single inheritance paradigm.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT) licensed.
+
```

Comment:
- [ ] I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same content in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**